index.html

```diff
@@ -0,0 +1,107 @@
+  </head>
+  <body>
+    <header>
+      <nav id="navbar">
+        <ul>
+          <li>
+            <a href="https://personal-portfolio.freecodecamp.rocks/#welcome-section">About</a>
+          </li>
+          <li>
+            <a href="https://personal-portfolio.freecodecamp.rocks/#welcome-section">Work</a>
+          </li>       
+          <li>
+            <a href="https://personal-portfolio.freecodecamp.rocks/#welcome-section">Contact</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section id="welcome-section" class="welcome">
+        <h1>Hey I am Mimic</h1>
+        <p>a web developer</p>
+      </section>
+
+      <section id="projects" class="flow">
+        <h1>These are some of my projects</h1>
+        <div id="projects-content" class="projects">
+          <div class="project"> 
+            <a href="https://codepen.io/freeCodeCamp/full/zNqgVx">
+              <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute.jpg" alt=""/>
+              <div>
+                <h3>Text</h3>
+              </div>
+            </a>
+          </div>
+          <div class="project">
+            <a href="https://codepen.io/freeCodeCamp/full/qRZeGZ">
+              <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/random-quote-machine.png" alt=""/>
+              <div>
+                <h3>Text</h3>
+              </div>
+            </a>
+            
+          </div>
+          <div class="project">
+            <a href="https://codepen.io/freeCodeCamp/full/wgGVVX">
+              <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/calc.png" alt=""/>
+              <div>
+                <h3>Text</h3>
+              </div>
+            </a>
+          </div>
+          <div class="project">
+            <a href="https://codepen.io/freeCodeCamp/full/mVEJag">
+              <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/map.jpg" alt=""/>
+              <div>
+                <h3>Text</h3>
+              </div>
+            </a>
+          </div>
+          <div class="project">
+            <a href="https://codepen.io/freeCodeCamp/full/wGqEga">
+              <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/wiki.png" alt=""/>
+              <div>
+                <h3>Text</h3>
+              </div>
+            </a>
+          </div>
+          <div class="project">
+            <a href="https://codepen.io/freeCodeCamp/full/KzXQgy">
+              <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tic-tac-toe.png" alt=""/>
+              <div>
+                <h3>Text</h3>
+              </div>
+            </a>
+          </div>
+         </div>
+         <div class="btn">
+          <a href="https://codepen.io/FreeCodeCamp/" id="show-all">Show All</a>
+         </div> 
+        </section>
+        <section id="work together" class="round-up">
+          <h1>Let's work together...</h1>
```

Comment:
- [OPTIONAL] Nice one @Trust-007, but try as much as you can to include `WHY` the student is asked to make use of a suggestion as this would make it clearer for the student to know why he or she is being asked to do a certain thing. And  if possible, you can also direct the student to a resource where more information could be got for  more clarification.